client/src/state.js

```diff
@@ -562,6 +562,8 @@ export const store = _createStore({
       if (resourceType !== undefined && resourceType !== "all") {
         rt = this.state.resourceTypeList.get(resourceType);
       }
+      if (exact == undefined)
+        exact = this.state.searchExactMatch;
```

Comment:
Does this default to True or false. can we set it with an environment variable

Reply:
Default is always False. 

I can try to set it with an environment variable but I think the default should be False. In this case, False means OR and that will get more search results.